lib/commentOnOpenedPullRequest.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+var Cesium = require('cesium');
+var Promise = require('bluebird');
+var requestPromise = require('request-promise');
+
+var dateLog = require('./dateLog');
+var checkStatus = require('./checkStatus');
+
+var Check = Cesium.Check;
+var defined = Cesium.defined;
+
+module.exports = commentOnOpenedPullRequest;
+
+/** Post comment on newly opened pull request
+ *
+ * @param {Object} jsonResponse Full response from GitHub event
+ * @param {Object} headers Headers to use in GET requests
+ * @param {String} changesSignature Message to post if CHANGES.md has changed in PR
+ * @param {String} thirdPartySignature Message to post if any third party files have changed
+ * @param {String[]} thirdPartyFolders Folder paths to search for third party file changes
+ * @return {Promise<http.IncomingMessage | String>} Response of POST
+ * @throws {DeveloperError} if jsonResponse or headers are not objects
+ */
+function commentOnOpenedPullRequest(jsonResponse, headers, changesSignature, thirdPartySignature, thirdPartyFolders) {
```

Comment:
Just hardcode `changesSignature` and `thirdPartySignature` into variables at the top of the file for now and we can expose them as configuration later.